src/main/resources/db/migration/common/V1_381__UnstructuredDataExpiryDeletionAutomated_task.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO darts.automated_task (aut_id,task_name,task_description,cron_expression,cron_editable, batch_size,
+created_ts, created_by,last_modified_ts,last_modified_by)
+VALUES (nextval('aut_seq'),'UnstructuredDataExpiryDeletion',
+ 'Deletes INBOUND, UNSTRUCTURED and ARM data that has reached its expiry date.',
```

Comment:
if its called `UnstructuredDataExpiryDeletion` it implies it only deleted from unstructured, not the other 2

Reply:
Agree with @cmagowan-hmcts. Unstructured is associated with one of the Data store in DARTS system. Since these all are associated objects may be "AssociatedObjectDataExpiryDeletionAutomatedTask" sounds better?

Reply:
Agreed updated :)